features/register.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Register
+  In order to sign up
+  As a user
+  I need to be able to fill in the data correctly and click Register
+
+  Scenario: Registration with correct data
```

Comment:
Otherwise at first test passes, but second time - error! Because this user is already in the database.

You should load fixtures after every scenario with @database or similar tag.